spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -122,137 +122,23 @@
-            should_proof_state_id: anything,
-            trace_id: subject.send(:amzn_trace_id),
-            threatmetrix_session_id: nil,
-            user_id: anything,
-            request_ip: request.remote_ip,
-            double_address_verification: false,
-          )
-
-        put :update
-      end
-
-      it 'only enqueues a job once' do
-        put :update
-        expect_any_instance_of(Idv::Agent).to_not receive(:proof_resolution)
-
-        put :update
-      end
-
-      context 'when pii_from_user is blank' do
-        it 'redirects' do
-          flow_session[:pii_from_user] = {}
-          put :update
-          expect(response.status).to eq 302
-        end
-      end
-
-      context 'when different users use the same SSN within the same timeframe' do
-        let(:user2) { create(:user) }
-
-        before do
-          allow(IdentityConfig.store).to receive(:proof_ssn_max_attempts).and_return(3)
-          allow(IdentityConfig.store).to receive(:proof_ssn_max_attempt_window_in_minutes).
-            and_return(10)
-        end
-
-        it 'rate limits them all' do
-          put :update
-          subject.idv_session.verify_info_step_document_capture_session_uuid = nil
-          put :update
-          subject.idv_session.verify_info_step_document_capture_session_uuid = nil
-          put :update
-          put :update
-          expect_any_instance_of(Idv::Agent).to_not receive(:proof_resolution)
-          expect(response).to redirect_to(idv_session_errors_ssn_failure_url)
-          subject.idv_session.verify_info_step_document_capture_session_uuid = nil
-
-          stub_sign_in(user2)
-          put :update
-          expect_any_instance_of(Idv::Agent).to_not receive(:proof_resolution)
-          expect(response).to redirect_to(idv_session_errors_ssn_failure_url)
-        end
-      end
-    end
-
-    context 'double address verification is enabled' do
-      let(:pii_from_user) { Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_ADDRESS.dup }
-      let(:capture_secondary_id_enabled) { true }
-      let(:enrollment) { InPersonEnrollment.new(capture_secondary_id_enabled:) }
-      before do
-        allow(user).to receive(:establishing_in_person_enrollment).and_return(enrollment)
-      end
-      it 'captures state id address fields in the pii' do
-        expect(Idv::Agent).to receive(:new).
-          with(Idp::Constants::MOCK_IDV_APPLICANT_STATE_ID_ADDRESS.merge(uuid_prefix: nil)).
-          and_call_original
-        put :update
-      end
-
-      it 'indicates to the IDV agent that double_address_verification is enabled' do
-        expect_any_instance_of(Idv::Agent).to receive(:proof_resolution).with(
-          kind_of(DocumentCaptureSession),
-          should_proof_state_id: anything,
-          trace_id: anything,
-          threatmetrix_session_id: anything,
-          user_id: anything,
-          request_ip: anything,
-          double_address_verification: true,
```

Comment:
Per our discussion, the verify info step still needs to pass a flag indicating in-person proofing at least. We can defer renaming from `double_address_verification` (to reduce refactoring scope), though, as long as the database field and feature flag usage are removed.